docs/HowToUseSmartFS.md

```diff
@@ -0,0 +1,139 @@
+# How to use SmartFS
+
+SmartFS is FileSystem used for user partition in TizenRT.
+
+SmartFS stands for Sector Mapped Allocation for Really Tiny (SMART) flash.
+It is a filesystem that has been designed to work primary with small,
```

Comment:
If you want to put these lines at different line, you should add two spaces at end of lines.
These are written in same line at markdown view, but you wrote them at different lines.

Reply:
Ok, It needs to be at different line. I will modify it . Thanks